Add conversation agent selector documentation (#28077)
home-assistant/home-assistant.github.io: source/_docs/blueprint/selectors.markdown

The output of this selector is the configured value when the toggle is on, it has not output otherwise.

## Conversation agent selector

> The conversation agent selector allows to pick a conversation agent.

Reviewer: ```suggestion
The conversation agent selector allows picking a conversation agent.
```

![Screenshot of a conversation agent selector](/images/blueprints/selector-conversation-agent.png)

> The selector has 1 option, `language`. This filters the conversation agents shown to the agents that support a specific lamnguage.

Reviewer: ```suggestion
The selector has 1 option, `language`. This filters the conversation agents shown, depending on the language.
```

```yaml
conversation_agent:
  language: en
```

{% configuration conversation_agent %}
language:

>   description: Limits the list of selectable conversations to agents that support the specified language.

Reviewer: ```suggestion
  description: Limits the list of conversation agents to those supporting the specified language.
```
